src/onemod/config/model_config/rover_config.py

```diff
@@ -41,6 +69,12 @@ class RoverConfig(ModelConfig):
 
     """
 
+    model_type: Literal["binomial", "gaussian", "poisson"] | None = None
+    observation_column: str | None = None
+    weights_column: str | None = None
+    train_column: str | None = None
+    holdout_columns: set[str] | None = None
+    coef_bounds: dict[str, tuple[float, float]] | None = None
     cov_exploring: set[str]
```

Comment:
`cov_exploring` seems not to be listed as explicitly required but has no default here. Is this fine? I guess attributes to Config classes are not the same as config settings, but just want to confirm

Reply:
So there are two kinds of required attributes: more general settings that can be inherited from the pipeline that have a `None` default and are included in `_required` (`add_pipeline_config()` will throw an error if not included in either stage or pipeline config), and stage-specific settings without a default (Pydantic will throw an error when initializing stage instance if not included)

We could always add the stage-specific attributes to `_required` to be less confusing, though the check in `add_pipeline_config()` would be unnecessary for them because Pydantic won't let them be unset. Open to other ways of handling these if we come up with other ideas!

Reply:
Gotcha, I don't think it will be a problem so long as the docstrings + out-of-code documentation make sense, and I don't have better ideas at the moment